src/extension.ts

```diff
@@ -55,6 +56,10 @@ let initializationError: ResponseError<InitializeError> | Error | undefined = un
 let crashCount = 0;
 
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
+  if (config('terraform').get<string>('trace.server') === 'verbose') {
```

Comment:
So the `trace.server` part is automatically added as part of the LSP client library, and produces pretty printed JSON request/responses.

I am taking advantage of that being always there to indicate we're in a debug/test context and not in production code. This can be added to the `.vscode/settings.json` file in the repo, and everything works automatically.

However this setting can be used in a bundled vsix, so it can be accidentally/purposefully enabled when we don't want it to be.

Honestly I was more focused on getting everything ported and working and less on the ergonomics. I'll think on this